stable/chronograf/README.md

```diff
@@ -0,0 +1,95 @@
+# Chronograf
+
+##  An Open-Source Time Series Visualization Tool
+
+[Chronograf](https://github.com/influxdata/chronograf) is an open-source web application built by the folks over at [InfluxData](https://influxdata.com) and written in Go and React.js that provides the tools to visualize your monitoring data and easily create alerting and automation rules.
+
+## QuickStart
+
+```bash
+$ helm install stable/chronograf --name foo --namespace bar
+```
+
+## Introduction
+
+This chart bootstraps A Chronograf deployment and service on a Kubernetes cluster using the Helm Package manager.
+
+## Prerequisites
+
+- Kubernetes 1.4+
+- PV provisioner support in the underlying infrastructure (optional)
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install --name my-release stable/chronograf
+```
+
+The command deploys Chronograf on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+$ helm delete my-release --purge
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following tables lists the configurable parameters of the Chronograf chart and 
+their descriptions. The default values can be seen in `values.yaml`:
```

Comment:
@lachie83 @prydonius 

Can we just start having comments in the values.yaml rather than duplicating information here too?  I did that over in #200.